src/docs/introduction/getting-started-with-ftw-daily/index.md

```diff
@@ -258,32 +258,34 @@ to learn how to apply for Stripe Connect review.
 **Other countries**<br /> If you're in any another country, follow these
 instructions to enable Stripe Connect:
 
-1. Click the **Connect** left menu item.
+1. Click the **Connect** top menu item, and then click the **Get
+   Started** button.
 
    ![Stripe connect](./stripe-connect.png)
 
-1. Click the **Get started** button.
+2. Once a modal opens, select **Platform or marketplace** and click
+   **Continue**.
 
-   ![Get started](./stripe-get-started.png)
+   ![Activate Stripe Connect](./stripe-connect-activate.png)
 
-1. After a few seconds, you should see your Connect dashboard. From the
-   left sidebar, go to **Settings** then **Connect settings**, at
-   https://dashboard.stripe.com/settings
+3. Now when you click the gear icon on the top bar to go to **Settings**
+   at https://dashboard.stripe.com/settings, you will see a new
+   **Connect** section. Click **Settings** in that section.
 
-   ![Connect settings](./stripe-connect-settings.jpg)
+![Connect settings](./stripe-connect-settings.png)
 
-1. Make sure that **Custom** is enabled in the **Account types** section
+3. Make sure that **Custom** is enabled in the **Account types** section
```

Comment:
```suggestion
4. Make sure that **Custom** is enabled in the **Account types** section
```

Or, with Markdown ordered lists, all entries can have entry number `1.`. Markdown will render the correct number and then there's no need to manually update the numbering if the entries change. Like it was in this list before.

Reply:
Fixed to entry 1 for all entries